public/locales/uk/pages.json

```diff
@@ -23,5 +23,47 @@
     "Why was my account suspended?": "Чому мій обліковий запис було призупинено?",
     "Why the name Trustroots?": "Чому Trustroots має таку назву?",
     "Why is there no forum?": "Чому немає форуму?",
-    "Why did you rename \"tribes\" to \"circles\"?": "Чому ви перейменували \"племена\" в \"кола\"?"
+    "Why did you rename \"tribes\" to \"circles\"?": "Чому ви перейменували \"племена\" в \"кола\"?",
+    "Host": "Хост",
+    "Help us build Trustroots! Nobody can do everything, but everyone can do something…": "Допоможіть нам збудувати Trustroots! Жоден не може зробити все, але кожен може зробити щось …",
+    "Help us build Trustroots!": "Допоможи нам збудувати Trustroots!",
+    "Great! Read about <2>development</2> and get in touch with us! We are looking for JavaScript developers (back- and frontend) and sysops.": "Чудово! Прочитай про <2> розробку </2> та зв’яжіться з нами! Ми шукаємо розробників JavaScript (back- та frontend) та sysops.",
+    "Gravatar, for profile images": "Gravatar, для зображення профілю",
+    "Get active!": "Ставай активним!",
+    "Frequently Asked Questions": "Поширені питання",
+    "Freedom": "Свобода",
+    "Free and open source": "Безкоштовний та з відкритим кодом",
+    "For developers": "Для розробників",
+    "Follow Trustroots": "Стежити за Trustroots",
+    "Find members": "Знайти користувачів",
+    "Files": "Файли",
+    "Facebook page": "Сторінка Facebook",
+    "Facebook group": "Facebook група",
+    "Facebook (Page)": "Facebook (Сторінка)",
+    "Facebook (Group)": "Facebook (Група)",
```

Comment:
> "Facebook page": "Сторінка Facebook",
> "Facebook group": "Facebook група",
> "Facebook (Page)": "Facebook (Сторінка)",
> "Facebook (Group)": "Facebook (Група)",

We should probably just combine these in source-code, even if it isn't big deal to translate double. 🙃  cc @hlibbabii